src/core/Wizard/Step.tsx

```diff
@@ -79,52 +77,23 @@ export const Step = (props: StepProps) => {
         ...style,
       }}
       onClick={onCompletedClick}
```

Comment:
what to do with workflow with this one...

Reply:
Moved `type !== 'workflow'` into `isPast` and `isActive` so `onClick` will not fire now